api/models/fixtures/cron_labels.yml

```diff
@@ -7,4 +7,4 @@ main:
 really-long-process-type-name:
   build: .
   labels:
-    - convox.cron.really-long-cron-job-name=0 * * * ? bin/myjob
```

Comment:
Hmm there was some existing validation, but we were allowing dashes:

https://github.com/convox/rack/blob/master/manifest/manifest.go#L93-L111